build.sbt

```diff
@@ -11,6 +11,8 @@ resolvers += Resolver.sonatypeRepo("snapshots")
 
 val scalafixCommonSettings = inConfig(IntegrationTest)(scalafixConfigSettings(IntegrationTest))
 
+bloopExportJarClassifiers in Global := Some(Set("sources"))
```

Comment:
This is necessary for IntelliJ to automatically download & attach sources for libraries